### DISCUSSION TITLE
Text does not appear.

### DISCUSSION THREAD

**Discussion by Playgameno0000** [dialogic-godot/dialogic#918] [Q&A]
I'm not good at English
This is the first time I install Dialogic, also I'm new to Godot so I'm not sure if this is a bug or I made a mess of it .
When i type the text it doesn't show

![Ảnh chụp màn hình (142)](https://user-images.githubusercontent.com/107024963/172310699-6a4a406d-11f9-4e43-a51d-283dcf2e2368.png)

**Comment by Yotopihi-Dev** [comment]
First, go to your theme editor and see if anything strange is going on there, next, go to the dialogic settings and re-insert the default theme, it's a bit buggy and I've had some issues when I didnt re-insert it

**Reply by Playgameno0000** [reply]
Thank but it doesn't work  
It only shows the ones that I wrote, but the new one still fails

**Comment by coppolaemilio** [comment]
Could it be the editor rendering size? We haven't optimized it for anything less than 100%
